### PR TITLE
Ui review changes

### PR DESCRIPTION
**Pull request by esoliani**
changes from Phil's review 
changes from Llanco's feedback

### CODE REVIEW

File: app/scripts/directives/gaugechart.coffee [hunk 1]
```diff
@@ -21,6 +21,7 @@ angular.module 'edudashAppDir'
         when value > ranges[1] then colors[2]
         else colors[1]
 
+      width = element.parent().width()
```
Comment (uniphil) [inline app/scripts/directives/gaugechart.coffee:24]: nice

File: app/scripts/directives/gaugechart.coffee [hunk 2]
```diff
@@ -29,16 +30,22 @@ angular.module 'edudashAppDir'
...
           plotShadow: false
           height: 110
-          width: 160
+          width: width
           margin: [0, 0, 0, 0]
-          marginLeft: 0
+          spacingLeft: 1
+        credits:
+          enabled: false
+        credits:
```
Comment (uniphil) [inline app/scripts/directives/gaugechart.coffee]: duplicate key `credits`?

File: app/views/ratepasschange.html [hunk 3]
```diff
@@ -1,5 +1,9 @@
 <div>
-    <p ng-class="{'text-green': data >= range[1], 'text-yellow': data < range[1] && data > range[0], 'text-red':data <= range[0]}"
-       style="font-size: 35px;font-weight: 900;margin-top: 0px; margin-bottom: 2px;margin-right: 10px">{{sign}}{{ data }}%</p>
-    <p style="color:#05a2dc;font-size:11;margin-bottom:2;fill:#05a2dc;width:96px;">{{'chart.dial-title'| translate}} {{ since }}</p>
+    <div class="col-xs-3 col-sm-4 col-md-12" style="height: 64px">
+        <p ng-class="{'text-green': data >= max, 'text-yellow': data < max && data > min, 'text-red':data <= min}"
+           style="font-size: 36px;font-weight: 900;margin-top: 0px; margin-bottom: 2px;margin-right: 10px">{{sign}}{{ data }}%</p>
```
Comment (uniphil) [inline app/views/ratepasschange.html]: can these inline styles move to the stylesheet?

File: app/scripts/directives/gaugechart.coffee (hunk 2 above)
Comment (esoliani) [inline app/scripts/directives/gaugechart.coffee]: remove one

File: app/views/ratepasschange.html (hunk 3 above)
Comment (esoliani) [inline app/views/ratepasschange.html]: create new css class

**Pull request merged**